portainer/app/executor.py

```diff
@@ -227,7 +227,7 @@ def _build_image(self, driver, taskInfo, buildTask):
                             str("%s: %s" % (image_name, message))
                         )
             else:
-                sandbox_dir = os.environ["MESOS_DIRECTORY"]
+                sandbox_dir = os.environ.get("MESOS_SANDBOX", os.environ["MESOS_DIRECTORY"])
```

Comment:
This makes me sad :disappointed:. I used `MESOS_DIRECTORY` to be consistent with the [built-in mesos containerizer](https://github.com/apache/mesos/blob/6b675dd1b2d8a392b1c6600266aacacf1378ff4e/src/slave/containerizer/containerizer.cpp#L284) but it turns out [the docker containerizer](https://github.com/apache/mesos/blob/6b675dd1b2d8a392b1c6600266aacacf1378ff4e/src/docker/docker.cpp#L332) is doing it's own thing. Wat.